index.html

```diff
@@ -112,159 +111,615 @@ <h1><a href="/">Latest funding rounds on Index.co ></a></h1>
+                        <li><a href="/">Robinhood is planning to go public — here's what we know</a></li>
+                        <li><a href="/">Fiat's 'sustainable driving' cryptocurrency is utter bollocks</a></li>
+                        <li><a href="/">Microsoft reportedly wants to buy Discord for more than $10B</a></li>
+                    </ul>
+                </div>
+            </div>
+
+            <div class="hard-fork-container">
+                <a class="hard-fork-container-tech" href="/">
+                    <h1>Podium ></h1>
+                </a>
+                <div class="hard-fork-1">
+                    <p class="span-1">MOST POPULAR</p>
+                    <a class="a-1" href="/">
+                        <h3>Intel's new AI helps you get just the right amount <br>of hate speech in your game chat</h3>
+                    </a>
+                    <p class="span-a"><a href="/">Ivan Mehta</a>- 1 day ago</p>
+                    <div class="hard-fork-latest">
+                        <img src="images/coinbase.jpg" alt="coinbase" class="hard-fork-image-22">
+                        <p class="hard-fork-p-a-1">LATEST</p>
+                        <a class="hard-fork-p-a-2" href="/">
+                            <p>What we know about Coinbase’s<br>April 14 IPO</p>
+                        </a>
+                    </div>
+                </div>
+                <div class="hard-fork-2">
+                    <ul>
+                        <li><a href="/">Digital art NFTs are all the rage — but they're horrible for Mother Earth</a>
                         </li>
+                        <li><a href="/">5 tips for an ethical investment in tech stocks</a></li>
+                        <li><a href="/">Robinhood is planning to go public — here's what we know</a></li>
+                        <li><a href="/">Fiat's 'sustainable driving' cryptocurrency is utter bollocks</a></li>
+                        <li><a href="/">Microsoft reportedly wants to buy Discord for more than $10B</a></li>
                     </ul>
                 </div>
             </div>
-        </nav>
+
+
+        </section>
+        <!--latest-deals-section-->
+        <section class="latest-deals">
+            <div class="latest-deals-header">
+                <a href="/">
+                    <h1>Latest deals ></h1>
+                </a>
+                <a href="/">
+                    <p>See all Deals</p>
+                </a>
+            </div>
+            <div class="latest-deals-container">
+                <div class="latest-deals-1">
+                    <div class="latest-deals-container-image">
+                        <div class="latest-deals-container-image-yellow">
+                            <p>98% OFF</p>
+                        </div>
+                        <div>
+                            <img src="images/python.jpg" alt="python" class="latest-deals-image-size">
+                        </div>
```

Comment:
change this div-tag to a-tag as the image needs to have link. Apply the same change for the other images in the latest-deals-container-image classes